### PR TITLE
Check for unique registry for ta

### PR DESCRIPTION
**Pull request by prmittal**

### CODE REVIEW

**Comment by prmittal** [comment]
@nigurr is added to the review. #Closed

File: Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1 [hunk 1]
```diff
@@ -30,7 +30,7 @@ function Locate-TestVersion()
 	{
 		$regPath = "HKLM:\SOFTWARE\Wow6432Node\Microsoft\DevDiv\vstf\Servicing"
 	}
-	$keys = Get-Item $regPath | %{$_.GetSubKeyNames()}
```
Comment (acesiddhu) [inline Tasks/DeployVisualStudioTestAgent/TestAgentConfiguration.ps1]: is this tested with existing dev14 test agents?

**Comment by acesiddhu** [comment]
> Check-TestAgentIsRunning -version "14.0"

why is this hard coded?

---
Refers to: Tasks/DeployVisualStudioTestAgent/VerifyTestMachinesAreInUse.ps1:67 in 519fbf1. [](commit_id = 519fbf127a5c72a22c1cb277dfd20d684d257309, deletion_comment = False)

**Comment by acesiddhu** [comment]
:shipit:

**Pull request merged**